source/reporting/index.html.md.erb

```diff
@@ -19,24 +19,21 @@ transaction information in a CSV file.
 
 You can export one file for one service's transactions, or one file for all your services' transactions.
 
-### Export one file for one service's transactions
+1. Go to [My Services](https://selfservice.payments.service.gov.uk/my-services).
 
-1. Select the account you want reporting information for in the [My Services](https://selfservice.payments.service.gov.uk/my-services) section.
+1. If you want to export one service's transactions, select the service account you want reporting information for and go to the [transactions](https://selfservice.payments.service.gov.uk/transactions) list.
 
-2. Go to the [transactions](https://selfservice.payments.service.gov.uk/transactions) list.
+    Select __View transactions for all live services__ if you want to export all your services' transactions.
 
-3. Filter transactions using the available search criteria.
+1. Filter transactions using the available search criteria if needed.
 
-4. Select __Download all transaction details as a CSV file__.
+    If you have more than 5000 transactions, you must apply a filter before the page will display your transactions.
```

Comment:
This is for CSV download link. If a service has less than 5K transactions, CSV download link is available immediately. Otherwise, a filter must be applied for CSV download link to appear.

Transactions are always displayed when services visit transactions page. Users can apply filters to refine transactions as needed